models/marts/enriched_history_operations.sql

```diff
@@ -188,6 +192,14 @@ with
             , batch_id
             , batch_run_date
             , batch_insert_ts
+            , asset_balance_changes
+            , parameters
+            , `function`
```

Comment:
are we missing function type? Which I know is an overloaded field at this point. Talking about the field with domain values `invoke_contract`, `restore_footprint`, `upload_wasm` etc